panel/_param.py

```diff
@@ -64,18 +60,13 @@ def _validate_value(self, val, allow_None):
         if val is None and allow_None:
             return
         if not isinstance(val, (tuple, int)):
-            raise ValueError(
-                f'Margin parameter {self.name!r} only takes integer and '
-                f'tuple values, not values of not {type(val)!r}.'
-            )
+            raise ValueError(f"Margin parameter {self.name!r} only takes integer and " f"tuple values, not values of not {type(val)!r}.")
 
     def _validate_length(self, val):
         if not isinstance(val, tuple) or len(val) in (2, 4):
             return
         raise ValueError(
-            f'Margin parameter {self.name!r} only takes integer and '
-            f'tuple values of length 2 (vertical, horizontal) or 4 '
-            '(top, right, bottom, left).'
+            f"Margin parameter {self.name!r} only takes integer and " f"tuple values of length 2 (vertical, horizontal) or 4 " "(top, right, bottom, left)."
```

Comment:
Hmm, this really does not make me trust the formatter.

Reply:
It looks weird, but isn't it still a valid code?

I think this happens because of the Panel's `line-length` of  160 (?)

Reply:
Yes valid, but I would expect the formatter to correctly join and then wrap the strings.

Reply:
For me `black` creates issues like these and `pylint` identifies them. Them I fix and don't see them again.